data/luminr-co.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Luminr",
+    "establishedAt": 1697548800000,
```

Comment:
This timestamp is for `October 17th 2023`. The field is the company established date

Reply:
Changed to the correct date